### ISSUE TITLE
master:  git submodule update with mavlink problems

### ISSUE THREAD

**Issue by xianglunkai** [ArduPilot/ardupilot#23889]
When synchronizing today's latest code, it was found that:
modified:   modules/mavlink (modified content)
Is there a .git file in mavlink code?

**Comment by magicrub** [comment]
Yeah, something weird happened to machine submodule recently. Not sure what it is.

**Comment by xianglunkai** [comment]
@magicrub 
Txs! I think this prx is found also in https://github.com/ArduPilot/ardupilot/issues/23902
